WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -1057,6 +1057,16 @@ extension WooAnalyticsEvent {
             )
         }
 
+        static func cardReaderAutomaticDisconnect(cardReaderModel: String?, forGatewayID: String?, countryCode: String) -> WooAnalyticsEvent {
+            WooAnalyticsEvent(statName: .cardReaderAutomaticDisconnect,
+                              properties: [
+                                Keys.cardReaderModel: readerModel(for: cardReaderModel),
+                                Keys.countryCode: countryCode,
+                                Keys.gatewayID: gatewayID(forGatewayID: forGatewayID)
```

Comment:
I didn't get "WooCommerce Payments" parsed through the gateway ID, but "unknown" instead:

```
2023-03-20 14:30:42.112488+0700 WooCommerce[71653:13550443] 🔵 Tracked card_reader_automatic_disconnect, properties: [AnyHashable("plugin_slug"): "unknown", AnyHashable("country"): "US", AnyHashable("card_reader_model"): "CHIPPER_2X", AnyHashable("is_wpcom_store"): false, AnyHashable("blog_id"): 215063064]
```

I see we're already injecting this value from the `SetUpTapToPayViewModelsOrderedList` via the `CardPresentPaymentsPlugin` in the constructor and there's only two possibilities (either wcpay or stripe), so I'm not sure if this is expected, I'm assuming the "unknown" value comes from this property being nil? Or perhaps because TTP? 🤔